bin/extract_codeobject

```diff
@@ -0,0 +1,426 @@
+#!/usr/bin/perl
+use strict;
+use File::Copy;
+use File::Spec;
+use File::Basename;
+use File::Which;
+use Cwd 'realpath';
+use Getopt::Std;
+use List::Util qw(max);
+
+sub usage {
+  print("Usage: $0 [OPTION]... -i <input>\n");
+  print("Extract the device kernels from an hcc executable.\n\n");
+  print("-h                                \tshow this help message\n");
+  print("-i <input>                        \t[DEPRECATED] input file\n");
+  print("-E <kernel-name>[:<kernel-name>]  \tExtract code object by URI name. Multiple code object names may be specified using \':\' delimiter.\n");
+  print("-L                                \tList code object bundles, by entry name, and their embedded kernel code objects, by URI, available in source input file (-i option).\n");
```

Comment:
List the bundled code objects, and the code objects they contain, that are embedded in the host executable specified by -e.

Would -l be a better name as easier to type and fits other commands that list things?